### ISSUE TITLE
[Bug]:  various game not reaching the title screen.

### ISSUE THREAD

**Issue by MelchiorGaspar** [project64/project64#2411]
### Expected behavior?

all games I for testing should be fully compatible with PJ64.

### Actual behavior?

I get a blank screen  after the logos and intro movie  for some.. no title screen and no sound...
odd as they all use to work.

### Steps to reproduce the problem

1. load latest build of 32 bit PJ64 and try to run any game I have.
2.
3.
4.
5.
...

[PJ64 Config.zip](https://github.com/project64/project64/files/14454318/PJ64.Config.zip)

### Error messages?

n/a

### What game were you playing?

Title: various titles I have.. for testing.

Nintendo 64 game or Nintendo 64 Disk Drive: game/ROM

ROM information (First you must uncheck "hide advanced settings" in the settings menu, then File > ROM info should show up, post the information contained there in this field. Alternatively, you can right click the ROM in the ROM browser and hit "ROM information".): 
Save type: (attach game save or save state here)

Original or modified ROM?:(attach patch if applicable here) original ROMs

### Specifications and build details

Project64: (include commit version number and whether you are using the 32-bit or 64-bit version): 
Dev-4.0.0-6350-290040d 32bit version.

OS: (include version and OS build number using WinVer): Windows 11 Pro v23H2 (build 22631.3235)

CPU: AMD Ryzen 9 3900X 12 cores / 24 threads.

RAM: 32 GB DDR4

Graphics: EVGA GeForce RTX 3070 Ti 8GB
Nvidia display driver v551.61

### Using included plugins

- [X] I can confirm I am using the included RSP, audio, input, and graphics plugins

### Plugins used while the issue occurred

![image](https://github.com/project64/project64/assets/1712842/b7593125-d20f-4d38-9d0a-51d09d401515)
the included audio plugins caused more rapid emulation speed problem.
when I tried to use Project64-Audio (v4.0.0.6350).dll

speed seems normal with [ AziAudioNEW (v0.70)(WIP10)(2017-12-27)(DS8 & XA2).dll ]

### Copyright Compliance

- [x] There are no infringing files attached to this bug report.

**Comment by Squall-Leonhart** [comment]
![image](https://github.com/project64/project64/assets/1098176/714fc1b4-9e7c-4e69-b3a6-2c2b261636ea)

**Comment by Squall-Leonhart** [comment]
oh, its just more RSP issues.

**Comment by MelchiorGaspar** [comment]
RSP plugin issues huh?
alrighty then.

**Comment by Squall-Leonhart** [comment]
Yeah, swap the RSP to interpreter, or Parallel-RSP if your pc can't handle the pj64 one in interpreter, though from  your list of games i only had a memory action error from goldeneye and the rest could reach menu's on recompiler here.

**Comment by Squall-Leonhart** [comment]
Aidyn chronicles wasn't RSP, its the 32bit engine option, this also likely applies to the Zelda games which fail to boot with it on atm too, already covered in #2300

**Comment by MelchiorGaspar** [comment]
yeah, I set Aidyn Chronicles - The First Mage (U) [!] (V1.1)'s per game settings to Interpreter for CPU style...
and that got the main menu to display for that game.
If I leave it set to recompiler but disable the 32bit engine setting.. it works then too..
for that game..

**Comment by LuismaSP89** [comment]
Regarding Goldeneye, the latest working build was Dev-4.0.0-6213-002f2e1 from 7th september 2023. Starting at build Dev-4.0.0-6224-f3d6d3f of 14th september it went broke.

The changelog for this broken version was:

Core: DisplayControlRegHandler::Read32 read more of the registers (commit: [9ffd87168a74a5ed6a884634bc65d2edc36d83bf](https://github.com/project64/project64/commit/9ffd87168a74a5ed6a884634bc65d2edc36d83bf))
    Core: Change COP0 Status register to a struct breaking up the bits (commit: [fcd7257adc7dcf62128c6455adfed9bb42b71df0](https://github.com/project64/project64/commit/fcd7257adc7dcf62128c6455adfed9bb42b71df0))
    Core: Have CRegisters::DoTLBReadMiss set the target pipe line to jump, not directly modify the PC (commit: [5da5dab3c5a755bd47a81de27e62dcd38e9ec6c1](https://github.com/project64/project64/commit/5da5dab3c5a755bd47a81de27e62dcd38e9ec6c1))
    Core: Add calls to CPO1_UNIMPLEMENTED_OP for Cop1.w functions (commit: [2d09178449a4548c97ef7b449edef25e14875d82](https://github.com/project64/project64/commit/2d09178449a4548c97ef7b449edef25e14875d82))
    Core: Have CRegisters::DoAddressError to not directly modify program counter (commit: [a5a4873e84e330ba1923335399502e7fd3e82cca](https://github.com/project64/project64/commit/a5a4873e84e330ba1923335399502e7fd3e82cca))
    Core: Move InitRegisters to register class (commit: [8b14b6d7d115035d35dfc29f361ded4abfeb1e8b](https://github.com/project64/project64/commit/8b14b6d7d115035d35dfc29f361ded4abfeb1e8b))
    Core: replace GenerateTLBReadException and void GenerateTLBWriteException with CRegisters::DoTLBReadMiss/CRegisters::DoTLBWriteMiss (commit: [ae4af8746bc76ca1019d5f805f2af487ec90fd9a](https://github.com/project64/project64/commit/ae4af8746bc76ca1019d5f805f2af487ec90fd9a))
    Core: Get CRegisters::DoAddressError, CRegisters::DoTLBReadMiss, CRegisters::DoTLBWriteMiss to use TriggerException function (commit: [f559aed2ad28b97489dab85ab90a52a804d78d2c](https://github.com/project64/project64/commit/f559aed2ad28b97489dab85ab90a52a804d78d2c))
    Core: Add LLD opcode (commit: [c02858c7a0692994aa7b7249c0b7b49fd5322338](https://github.com/project64/project64/commit/c02858c7a0692994aa7b7249c0b7b49fd5322338))
    Core: Fix clang issue (commit: [e0c125e8370a32e47c20f0a191a4d6a362acbd47](https://github.com/project64/project64/commit/e0c125e8370a32e47c20f0a191a4d6a362acbd47))
    Core: for tlb miss only use special address when address is not defined (commit: [f3d6d3fc7c5bd6ae759237d82e03532a2ac5f7ca](https://github.com/project64/project64/commit/f3d6d3fc7c5bd6ae759237d82e03532a2ac5f7ca))

**Comment by mrwallace888** [comment]
I just grabbed the latest nightly of Project 64, and using default settings most of the games were just a black screen with 0 FPS and didn't work. Changing video plugins and everything did nothing, as well as disabling HLE. It says the emulation loaded, but there's nothing there. Diddy Kong Racing only shows the logo for a moment and is just stuck with a black screen and a quiet high pitched noise. The most I was able to do was get Super Mario 64 to work, but even then, the emulation just froze as soon as Mario jumps out of the pipe at the start.

The logs directory has 3 log files but they're all 0 bytes and empty.

I wonder if my issues are related to this.

**Comment by pcca-matrix** [comment]
try disabling 32-bit mode and/or Advanced Block Linking.
However, it's best to stay away from recents vnightly builds and use the "latest valid" version, which is for me , as LuismaSP89 writes, and i confirm,  version 4.0.0-6213. 
Since this version, there haven't been any significant improvements , only regressions and some tweak that are useless for the end user (at least for now)
